crates/matrix-sdk-crypto/src/verification/mod.rs

```diff
@@ -634,9 +634,10 @@ impl IdentitiesBeingVerified {
                 if verified_identities.is_some_and(|i| {
                     i.iter().any(|verified| verified.user_id() == identity.user_id())
                 }) {
-                    trace!(
+                    info!(
```

Comment:
Does this have to be an `info!` level? I suppose it's fine because it shouldn't happen that often?

Reply:
I think it's kinda important -- about the same as other stuff that is logged at info -- and it should only happen once per verification.